dsl-reference.md

```diff
@@ -1416,6 +1417,36 @@ from:
 to: '.petList += [ . ]'
 ```
 
+### Export
+
+Certain task needs to set the context of the workflow using the task output for later usage. User set the content of the context through a runtime expression. The result of the expression is the new value of the context. If user want to merge the new data into the current context, he might do that using `$context` variable. 
+
```

Comment:
This is incorrect. Doing so would go against what was discussed.

The `export/as` expression is evaluated against the context. 
The user can access the input and the output of the task using `input` and `ouput` arguments.

Reply:
So . is the context, isnt it?

Reply:
In that case - **AND ONLY THAT CASE** - yes.

So you would have:

```yaml
do:
  call: http
  with: {...}
  export:
    as: . + { responseStatusCode = $output.statusCode }
```

Given the context looked like this before the export:

```yaml
foo:
  bar: baz
```

And given the task output's `statusCode` property being 503;

After evaluating the context it would look like:

```yaml
foo:
  bar: baz
responseStatusCode: 503
```

That is why I was so reticent to rename `export/to` into `export/as`, because the expression itself is doing a `to`.

Reply:
I do not think how to access the context value (. being $context rather than  . being $output) is really related with the to/as. 
I think we need to clearly set what . is supposed to be for every expr, probably we should open a PR to do that as result of dicussion https://github.com/serverlessworkflow/specification/discussions/890